src/Login/Signin/signin.spec.tsx

```diff
@@ -0,0 +1,63 @@
+import { render, screen, fireEvent, cleanup } from "@testing-library/react"
+import { SignIn } from "./signin"
+import {
+    BrowserRouter as Router,
+    Switch,
+    Route
+} from 'react-router-dom';
+
+function TestEnvironment(props: { children: any }) {
+    return <Router>
+        <Switch>
+            <Route>
+                {props.children}
+            </Route>
+        </Switch>
+    </Router>
+}
+
+describe('SignIn', () => {
+    beforeEach(() => {
+        return render(
+            <TestEnvironment>
+                <SignIn />
+            </TestEnvironment>
+        )
+    })
+    it('should render logo', () => {
+        const logo = screen.getByTestId("logoTest")
+        expect(logo).toHaveClass("logoPlace")
+    })
+    it("should render input field", () => {
+        screen.getAllByTestId('inputComp')
+    })
+    it("should render E-Mail field", () => {
+        screen.getByPlaceholderText('E-Mail')
+    })
+    it("should render Passwort field", () => {
+        screen.getByPlaceholderText('Passwort')
+    })
+    it('should render Login button', () => {
+        const loginButton = screen.getByTestId("SecondaryButton")
+        expect(loginButton).toHaveClass("SecondaryButton")
```

Comment:
toHaveClass muss eigentlich nur eingesetzt werden, wenn man die Änderung einer Klasse betrachten will. Hier ist die Klasse fest und es ist keine Änderung in Sicht.